src/components/learner-details.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import Modal from '@mui/material/Modal';
+import {
+  Box,
+  Button,
+  Divider,
+  Grid,
+  IconButton,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material';
+import { useTranslation } from 'next-i18next';
+import CloseIcon from '@mui/icons-material/Close';
+import PlaceOutlinedIcon from '@mui/icons-material/PlaceOutlined';
+import ProfileField from '@/components/ProfileField';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+
+const LearnerDetails = () => {
+  const { t } = useTranslation();
+
+  const [open, setOpen] = React.useState(true);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  const dummyData = [
+    { label: 'Full Name', value: 'Ananya Gupta' },
+    { label: 'Age', value: '19' },
+    { label: 'Gender', value: 'Female' },
+    { label: 'Type Of Learner', value: 'Regular' },
+    { label: 'Enrollment Number', value: '1002365362' },
+    { label: 'Learner Primary work', value: 'Self Employed' },
+    { label: 'contact Number', value: '945454665' },
+  ];
+
+  const theme = useTheme<any>();
+  const isDesktop = useMediaQuery(theme.breakpoints.up('md'));
+
+  const style = {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: isDesktop ? 500 : 400,
+    bgcolor: 'warning.A400',
+    p: 4,
+    textAlign: 'center',
+    height: 'auto',
+  };
+  return (
+    <div>
+      <Modal
+        open={open}
```

Comment:
@upendraTekdi there is modal component already created, please try to reuse same

Reply:
There have some issue in created modal component so need to implementation for that .

Reply:
we need some extra button for that reusable model